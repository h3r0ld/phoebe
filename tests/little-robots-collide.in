start map.txt
hardworking-little-robot-random off
robot-move 0 10 10
robot-move 1 10 10
quit
