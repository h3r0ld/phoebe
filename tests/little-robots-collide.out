Hardworking-little-robot 0: New Position: Cell(10, 10)
Hardworking-little-robot 1: New Position: Cell(10, 10)
Hardworking-little-robot 0: Collision; New Position: Cell(8, 10)
Hardworking-little-robot 0: Collision; New Position: Cell(12, 10)